config/jobs/kubernetes/sig-autoscaling/sig-autoscaling-presubmits.yaml

```diff
@@ -0,0 +1,48 @@
+presubmits:
+  kubernetes/kubernetes:
+  - name: pull-kubernetes-e2e-gci-gce-autoscaling
+    skip_branches:
+    - release-\d+\.\d+
+    annotations:
+      testgrid-dashboards: sig-autoscaling-cluster-autoscaler
+      testgrid-tab-name: gci-gce-autoscaling-pull
+      fork-per-release: "true"
+    always_run: false
+    optional: true
```

Comment:
@krzyzacy this is optional so I guess 🤷‍♂

Reply:
> Sen Lu this is optional so I guess

Does that mean that we should just merge it and see if that works afterwards?

Reply:
_er.
I think it's acceptable to add new presubmits that run sometimes and **are optional**, IE do not block merges. New presubmits should not block merges immediately especially since we don't know how well they work yet. Ideally this should have a well defined process but I'm not sure we have one yet :/

Reply:
I see. So (unless we have better idea - I do not - yet I do not have much context in the area) let's merge it. And I will send out a test PR to k8s/k8s which modifies `cluster/gce/manifests/cluster-autoscaler.manifest` or `cluster/addons/rbac/cluster-autoscaler` and see if presubmit triggers. And if there are issues lets iterate from there.

WDYT>

Reply:
@BenTheElder ping :)